conans/client/generators/cmake_find_package.py

```diff
@@ -58,8 +58,13 @@ def _find_for_dep(self, name, cpp_info):
             # Here we are generating FindXXX, so find_modules=True
             public_deps_names = [self.deps_build_info[dep].name for dep in cpp_info.public_deps]
             lines = find_dependency_lines(name, public_deps_names, find_modules=True)
+        # get {{name}}_INCLUDE_DIR as a first include path if exists
+        if cpp_info.include_paths:
+            include_dir = cpp_info.include_paths[0]
+        else:
+            include_dir = '""'
```

Comment:
What if include_dir is just empty instead of ""? The final var:

```cmake
set(ZLIB_INCLUDE_DIR)  # Good
#no need of 
set(ZLIB_INCLUDE_DIR "") 
```

Reply:
This may be not actual with a new commit that uses all include paths